python/pacemaker/_cts/scenarios.py

```diff
@@ -277,10 +277,6 @@ def summarize(self):
 
             self._cm.log(f"{f'Test {test.name}':<25} {stat_filter!r}")
 
-        self._cm.debug("Detailed Results")
-        for test in self.tests:
-            self._cm.debug(f"{f'Test {test.name}':<25} {stat_filter!r}")
```

Comment:
This was actually a regression in 8e8244c4. Detailed results used to log all of `test.stats` instead of `stat_filter`. Which makes a lot of sense.

Reply:
Fixed.